localcmd/local.go

```diff
@@ -78,3 +111,12 @@ func Run(args Args) (err error) {
 	}
 	return nil
 }
+
+func IsValidFormat(f Format) bool {
```

Comment:
This is actually called from the main package to validate the format so we can return an error to the user (so it might need to stay public if i'm not mistaken).

I could have moved the validation into main.go, but It felt like it shouldn't be aware of which formats are supported by local. Maybe that's the wrong way to look at it.